contributors/devel/e2e-tests.md

```diff
@@ -432,7 +431,7 @@ at a custom host directly:
 export KUBECONFIG=/path/to/kubeconfig
 export KUBE_MASTER_IP="127.0.0.1:<PORT>"
 export KUBE_MASTER=local
-go run hack/e2e.go -- -v --test
+go run hack/e2e.go -- -v --test --provider=local
```

Comment:
The option doesn't have the default value like:

test/e2e/framework/test_context.go
  219         flag.StringVar(&TestContext.Provider, "provider", "", "The name of the Kubernetes provider (gce, gke, local, vagrant, etc.)")

So we need the flag here.